gymnasium/spaces/box.py

```diff
@@ -25,12 +27,12 @@ def _short_repr(arr: np.ndarray) -> str:
     return str(arr)
 
 
-def is_float_integer(var) -> bool:
+def is_float_integer(var: Any) -> bool:
     """Checks if a variable is an integer or float."""
     return np.issubdtype(type(var), np.integer) or np.issubdtype(type(var), np.floating)
 
 
-class Box(Space[np.ndarray]):
+class Box(Space[NDArray[Any]]):
```

Comment:
I'd omit the `Any` in `NDArray[Any]`, it doesn't add any information and makes it less readable

Reply:
(and in most other places where it's `[Any]`)

Reply:
I think when I was testing with the strong pyright (including reportGeneralTypeIssues) then pyright was complaining saying that the Generic type was missing from `Space` and `npt.NDArray` so while it is obvious Im going to keep it for now